sandbox/data/custom-node/CustomNode.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
This should not be here you probably need to get your fork up to date with master